lte/cloud/go/services/subscriberdb/storage/digest_lookup_test.go

```diff
@@ -0,0 +1,105 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package storage_test
+
+import (
+	"testing"
+	"time"
+
+	"magma/lte/cloud/go/services/subscriberdb/storage"
+	"magma/orc8r/cloud/go/sqorc"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDigestLookup(t *testing.T) {
+	db, err := sqorc.Open("sqlite3", ":memory:")
+	assert.NoError(t, err)
+	s := storage.NewDigestLookup(db, sqorc.GetSqlBuilder())
+	assert.NoError(t, s.Initialize())
+
+	t.Run("empty initially", func(t *testing.T) {
+		digest, timestamp, err := storage.GetDigest(s, "n0")
+		assert.NoError(t, err)
+		assert.Equal(t, digest, "")
+		assert.Equal(t, timestamp, int64(0))
+
+		networkIDs, err := storage.GetAllNetworkIDs(s)
+		assert.NoError(t, err)
+		assert.Equal(t, []string{}, networkIDs)
+	})
+
+	t.Run("basic insert", func(t *testing.T) {
+		err = s.SetDigest("n0", "00000")
```

Comment:
Similar to a previous comment: use something like "apple", "banana", "cherry" instead of numbers -- basically, any alphabetized words you want